apis/nucleus/src/components/listbox/ListBoxRowColumn.jsx

```diff
@@ -230,46 +258,64 @@ export default function RowColumn({ index, style, data, column = false }) {
     padding: checkboxes ? 0 : undefined,
   };
 
-  return (
-    <Grid
-      container
-      spacing={0}
-      classes={{
-        root: classes.fieldRoot,
-      }}
-      className={joinClassNames(['value', ...classArr])}
-      style={style}
-      onClick={onClick}
-      onMouseDown={onMouseDown}
-      onMouseUp={onMouseUp}
-      onMouseEnter={onMouseEnter}
-      role={column ? 'column' : 'row'}
-      tabIndex={0}
-      data-n={cell && cell.qElemNumber}
-    >
-      <Grid item style={cellStyle} className={classes.cell} title={`${label}`}>
-        {ranges.length === 0 ? getField({ lbl: label, color: 'inherit' }) : getFieldWithRanges({ lbls: labels })}
-      </Grid>
+  const hasHistogramBar = () => cell && histogram && getFrequencyText() !== '-';
+  const getBarWidth = (qFrequency) => {
+    const freqStr = String(qFrequency);
+    const isPercent = freqStr.substring(freqStr.length - 1) === '%';
+    const freq = parseFloat(isPercent ? freqStr.slice(0, -1) : qFrequency);
```

Comment:
I think `parseFloat` works without manually removing `%`.
I.e. `parseFloat('20.2%') // => 20.2`

Reply:
Ahh, didn't know that